components/esp_hid/include/esp_hid_common.h

```diff
@@ -23,6 +23,11 @@ extern "C" {
 #include <stdbool.h>
 #include <stdio.h>
 
+/* HID BT Task Size Def */
```

Comment:
If you want to use the configuration option in Kconfig, code needs to be modified as:

```C
#include "sdkconfig.h"

#ifdef CONFIG_BT_HID_DEVICE_TASK_SIZE
#define BT_HID_DEVICE_TASK_SIZE            CONFIG_BT_HID_DEVICE_TASK_SIZE
#else
#define BT_HID_DEVICE_TASK_SIZE            4096
#endif

```